modules/sbo-projecting-the-binding-data.adoc

```diff
@@ -0,0 +1,17 @@
+[id="sbo-projecting-the-binding-data_{context}"]
+= Projecting the binding data
+
+Depending on your workload requirements and environment, you can choose to project the binding data either as files or environment variables.
+
+.Prerequisites
+
+* You have understood the following concepts:
```

Comment:
```suggestion
* You understand the following concepts:
```
Please review the IBM SG "Guidance for all tones" section, which includes "Use simple present tense, as much as possible."

Reply:
Will implement this.

Reply:
Will implement this.